<launch>
  <node name="ip_mapper_node" pkg="ip_mapping" type="ip_mapper_node" output="screen">
    <param name="camera_name"     type="string"   value="app/camera/rgb/image_raw"/>  
    <param name="ow"         	  type="int" 		  value="640"/>  
    <param name="oh"		   	  type="int" 		  value="240"/>


    <!-- jack 13 720p calibration: -->
    <!--param name="f_u"             type="double"   value="927.054138"/>
    <param name="f_v"             type="double"   value="935.428772"/>
    <param name="c_u"             type="double"   value="654.456376"/>
    <param name="c_v"             type="double"   value="346.241851"/-->
    

    <!-- jack 13 640*480 - cut to only bottom half (640*240) calibration: -->
    <param name="f_u"             type="double"   value="655.554626"/>
    <param name="f_v"             type="double"   value="652.052734"/>
    <param name="c_u"             type="double"   value="312.773367"/>
    <param name="c_v"             type="double"   value="7.779505"/>
    

    <param name="deg"             type="double"   value="8"/>
    <param name="cam_h"           type="double"   value="18"/>
  </node>

</launch>
